src/WebJobs.Script/Workers/SharedMemoryDataTransfer/SharedMemoryConstants.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
+{
+    internal class SharedMemoryConstants
+    {
+        /// <summary>
+        /// The length in number of bytes of a <see cref="long"/>.
+        /// It is used to specify the length (in the header) of content contained in a <see cref="SharedMemoryMap"/>.
+        /// </summary>
+        public const int LengthNumBytes = sizeof(long);
```

Comment:
Name is not very clear. MaxBytesToWrite?

Reply:
This is to define the number of bytes taken by a `long` which describes the length of content contained in the `MemoryMappedFile`, used in its header. So I have renamed to `ContentLengthHeaderBytes`.